tests/components/zwave/test_sensor.py

```diff
@@ -109,7 +111,7 @@ def test_multilevelsensor_value_changed_temp_celsius(hass, mock_openzwave):
     device.hass = hass
     assert device.state == 38.9
     assert device.unit_of_measurement == homeassistant.const.TEMP_CELSIUS
-    assert device.device_class == homeassistant.const.DEVICE_CLASS_TEMPERATURE
+    assert device.device_class == SensorDeviceClass.TEMPERATURE
```

Comment:
This is not state attributes, so you should be able to use `is` here

Reply:
Agreed, done.